Improve texts (#6)
postgres-ai/dle-github-actions: README.md

# Database Lab (Postgres.ai) to test DB migrations in CI

This GitHub action tests DB migrations database schema changes (database migrations) automatically using thin clones of large databases provided by Database Lab Engine (DLE)

## Overview
**Database Lab DB migration checker** is a tool to automatically test migrations in CI/CD pipelines.

## Key features
- Check migrations as a part of a standard CI/CD pipeline
- Automatically detect (and prevent!) long-lasting dangerous locks that could put your production systems down

> - Run all tests in secure environment: data cannot be copied to outside of a secured container

Reviewer: ```suggestion
- Run all tests in secure environment: data cannot be copied outside the secure container
```

> - Collect useful artifacts (such as `pg_stat_***` system views) and use them to enpower your DB changes review process

Reviewer: ```suggestion
- Collect useful artifacts (such as `pg_stat_***` system views) and use them to empower your DB changes review process
```
